rolling/distribution.yaml

```diff
@@ -2597,6 +2597,24 @@ repositories:
       url: https://github.com/swri-robotics/marti_messages.git
       version: ros2-devel
     status: developed
+  master_board_sdk:
+    doc:
+      type: git
+      url: https://github.com/open-dynamic-robot-initiative/master-board
+      version: master
+    release:
+      packages:
+      - master_board_sdk
+      tags:
+        release: release/rolling/{package}/{version}
+      url: https://github.com/ros2-gbp/master-board
```

Comment:
This repo doesn't exist yet. Please remove the release stanza and ask for it to be created after this is reviewed and merged via: https://github.com/ros2-gbp/ros2-gbp-github-org/blob/latest/CONTRIBUTING.md

Then bloom can fill in the release stanza.